changelog/0053-upgrade-redis.yml

```
@@ -0,0 +1,26 @@
+title: upgrade-redis
+key: upgrade-redis
+date: 2022-08-23
+optional_per_env: no
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version:
+# (optional) Max version of HQ that can be deployed before this change MUST be rolled out (commit hash)
+max_commcare_version:
+
+context: |
+  This change upgrade Redis from 4.0.8 to 6.x version.
+  As part of our ongoing effort to keep CommCare HQ up to date with the latest tools and
+  libraries we have updated Redis from version 4.0.8 to version 6.2.
+
+details: |
+  Redis is strongly backwards compatible, so no coordination with the CommCare HQ codebase is required before this upgrade.
+  However, after six weeks, from this announcement, new changes to CommCare HQ will no longer be guaranteed
+  to be compatible with Redis 4.0.8, so we strongly recommend applying this upgrade as soon as possible.
+
+update_steps: |
+  Before you start, make sure that you have backed up the existing Redis database servers
+  to a remote location.
+
+  0. For users running postgresql on Linux machines:
```

Should this say "redis" instead of "postgresql"?
```suggestion
  0. For users running Redis on Linux machines:
```

---

Apologies. I missed it. Correcting it.